dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporterTest.java

```diff
@@ -136,9 +141,519 @@ public void testAddPrefix() throws IOException {
+    DoubleSumData doubleSumData =
+        DoubleSumData.create(true, AggregationTemporality.CUMULATIVE, doublePointDataCollection);
+    MetricData metricData =
+        MetricData.createDoubleSum(
+            Resource.getDefault(),
+            InstrumentationLibraryInfo.empty(),
+            "doubleSumData",
+            "",
+            "",
+            doubleSumData);
+
+    DynatraceMetricExporter metricExporter =
+        DynatraceMetricExporter.builder()
+            .setApiToken("mytoken")
+            .setUrl(connection.getURL())
+            .build();
+
+    CompletableResultCode result =
+        metricExporter.export(Collections.singleton(metricData), connection);
+
+    verify(connection).setRequestMethod("POST");
+    verify(connection).setRequestProperty("Authorization", "Api-Token mytoken");
+    verify(connection).setRequestProperty("Content-Type", "text/plain; charset=utf-8");
+    assertEquals(
+        "doubleSumData,dt.metrics.source=opentelemetry count,delta=200.3 1619687659000\n"
+            + "doubleSumData,dt.metrics.source=opentelemetry count,delta=200.2",
+        bos.toString());
+    assertEquals(CompletableResultCode.ofSuccess(), result);
+  }
+
+  @Test
+  void testDoubleSumLinesDelta() throws IOException {
+    ByteArrayOutputStream bos = new ByteArrayOutputStream();
+    ByteArrayInputStream bis =
+        new ByteArrayInputStream(
+            "{\n\"linesOk\": 3,\n\"linesInvalid\": 0,\n  \"error\": null\n}".getBytes());
+
+    HttpURLConnection connection = setUpMockConnection(202, bos, bis);
+
+    Collection<DoublePointData> doublePointDataCollection =
+        new ArrayList<DoublePointData>() {
+          {
+            add(
+                DoublePointData.create(
+                    1619687639000000000L, 1619687659000000000L, Attributes.empty(), 123.456d));
+            add(
+                DoublePointData.create(
+                    1619687639000000000L, 1619687659000000000L, Attributes.empty(), 321.456d));
+            add(DoublePointData.create(0L, 0L, Attributes.empty(), 654.321d));
+          }
+        };
+    DoubleSumData doubleSumData =
+        DoubleSumData.create(true, AggregationTemporality.DELTA, doublePointDataCollection);
+    MetricData metricData =
+        MetricData.createDoubleSum(
+            Resource.getDefault(),
+            InstrumentationLibraryInfo.empty(),
+            "doubleSumData",
+            "",
+            "",
+            doubleSumData);
+
+    DynatraceMetricExporter metricExporter =
+        DynatraceMetricExporter.builder()
+            .setApiToken("mytoken")
+            .setUrl(connection.getURL())
+            .build();
+
+    CompletableResultCode result =
+        metricExporter.export(Collections.singleton(metricData), connection);
+
+    verify(connection).setRequestMethod("POST");
+    verify(connection).setRequestProperty("Authorization", "Api-Token mytoken");
+    verify(connection).setRequestProperty("Content-Type", "text/plain; charset=utf-8");
```

Comment:
```suggestion
    verifyConnectionProperties(connection);
```
i wonder if its worth putting these three lines into a separate function, they are repeated a lot, right?

Reply:
Yes. Done :)